test/missing-package/webpack-dev-server/missing-webpack-dev-server.test.js

```diff
@@ -0,0 +1,28 @@
+'use-strict';
+const { run } = require('../../utils/test-utils');
+const path = require('path');
+const { renameSync } = require('fs');
+
+const pathToPackage = require.resolve('webpack-dev-server').split(path.sep);
+const pathToBase = pathToPackage.slice(0, pathToPackage.indexOf('webpack-dev-server'));
+const pathToPacakge = (pkg) => {
+    return [...pathToBase, pkg].join(path.sep);
+};
+
+describe('missing webpack-dev-server', () => {
+    beforeEach(() => {
+        renameSync(pathToPacakge('webpack-dev-server'), pathToPacakge('.webpack-dev-server'));
```

Comment:
I am afraid it will not work, we need here smoke test

Reply:
Yup, going to do that and integrate it with nyc to catch coverage.